src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -507,7 +507,7 @@ public async Task VerifyBankAsync(Guid organizationId, int amount1, int amount2)
             await _paymentService.PurchaseOrganizationAsync(organization, signup.PaymentMethodType.Value,
```

Comment:
Non-blocking: might be nice to refactor this method to take an object created from the `signup` object/properties instead of the slow-n-steady bloat of method signature size.